ci-operator/config/openshift/kube-proxy/openshift-kube-proxy-master.yaml

```diff
@@ -0,0 +1,34 @@
+base_images:
+  base:
+    name: "4.1"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+canonical_go_repository: github.com/openshift/kube-proxy
```

Comment:
Hmm, this repo doesn't have any code in it so far though. Aren't we still creating the image from openshift/origin at htis point?

Reply:
Its from origin/images/kube-proxy